samples/breakaway-speech-button/public/index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <meta name="theme-color" content="#000000">
+    <!--
+      manifest.json provides metadata used when your web app is added to the
+      homescreen on Android. See https://developers.google.com/web/fundamentals/web-app-manifest/
```

Comment:
What's this? Is this from another template somewhere?

Reply:
Yes, this is done by `create-react-app`, the official + optional way for creating React app. Many people use it.

You don't really need `create-react-app` if you know Babel + Webpack (or if you don't need them). It's just conveniences for a lot of simple case. :)